spec/models/user_spec.rb

```diff
@@ -1,9 +1,89 @@
 require 'rails_helper'
 
 describe User do
+	
+	# does this slow performance?
+	before(:each) do
+    @user = create(:user)
+  end
 
   it { should belong_to :team }
   it { should belong_to :unit }
   it { should have_many :bills }
+  it { should have_many :invitations }
+  it { should have_many :senders }
 
+  it 'has a valid factory' do
+  	expect(@user).to be_valid
+  end
+  
+  describe '.phone' do
+  	it 'must have 10 characters' do
```

Comment:
Same note here. Shoulda-matchers may be able to reduce a lot of these examples to one-liners.